### ISSUE TITLE
Opening A Large Number of Files in a Project Steals Mouse and Keyboard Focus Constantly

### ISSUE THREAD

**Issue by utkonos** [Vector35/binaryninja-api#7819]
**Version and Platform (required):**
- Binary Ninja Version: 5.3.8813-dev Ultimate, 67daa910
- OS: macos
- OS Version: 26.2.0
- CPU Architecture: arm64

**Bug Description:**
If I have a large number of new files that I want to have opened at once in a project, there is a progress meter popup window that steals mouse and keyboard focus for each newly opened file. This prevents you from working in other applications while you wait for binja to grind out the set of analyses. If you have 20 or 50 files opening, you simply cannot work in other windows, your mouse focus just returns to binary ninja.

**Steps To Reproduce:**
1. Put 50 files in a project.
2. Select all of them
3. Right click
4. Open them all.
5. Try to click on any window anywhere that isn't binja.
6. Note that mouse focus gets snatched over and over and over for each new file opened.

**Expected Behavior:**
This should be a type of background task. The mouse and keyboard focus should remain where the user is working while this task completes.
